lib/bamboo/adapters/ses_adapter.ex

```diff
@@ -36,11 +38,7 @@ defmodule Bamboo.SesAdapter do
          |> put_html(email.html_body)
          |> put_attachments(email.attachments)
          |> Mail.render(RFC2822Renderer)
-         |> SES.send_raw_email(
-           configuration_set_name: configuration_set_name,
-           template: template,
-           template_data: template_data
-         )
+         |> SES.send_raw_email(send_raw_email_opts)
```

Comment:
Is `send_raw_email/1` really receive `template` option? 🤔 It was not in type.
https://hexdocs.pm/ex_aws_ses/2.1.1/ExAws.SES.html#send_raw_email/1
https://hexdocs.pm/ex_aws_ses/2.1.1/ExAws.SES.html#t:send_raw_email_opt/0